homeassistant/components/prowl/notify.py

```diff
@@ -48,6 +48,8 @@ async def async_send_message(self, message, **kwargs):
             "description": message,
             "priority": data["priority"] if data and "priority" in data else 0,
         }
+        if data.get("url") is not None:
+            payload["url"] = data.get("url")
```

Comment:
I know it's only a small PR I've submitted. Simple enough. Python is not my primary language.

I'm happy enough with line 52 (my first edit did that), but could the "is not None" removal perhaps allow a blank or NULL string to be sent?
I'd tested pretty much all eventualities manually, and wonder if this is a tidy for prettiness rather than functionality?
As I say, not my first language, and I'm reading and learning as I go. Also, a little late in the evening to test this.
Thought I'd just ask the question first before spinning up my test system again.
Thanks.

Reply:
It may be worth considering retrospectively building the correct tests for Prowl. Again, an area I'd be happy to help with, but learning as I go.

Reply:
If "" or None (null) if will not be true

Reply:
I'm confused @elyobelyob, you resolved the comment and said you were happy with the change, but you did not commit the suggestion?

Reply:
Oops, sorry. I did try one way, and it said that I couldn't as Author. But just found a different button to press.